content/en/docs/setup/getting-started/index.md

```diff
@@ -332,7 +332,7 @@ $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -
 
     {{< text bash >}}
     $ echo "$GATEWAY_URL"
-    192.168.99.100:32194
+    127.0.0.1:80
```

Comment:
Since we default to the Minikube tab for this page, this would make sense. Not that if you use another tab, the value will likely be different.